### PR TITLE
RFC Dynamic Idle

### PR DESCRIPTION
**Pull request by haslinghuis**
PR implementing https://github.com/betaflight/betaflight/pull/11043

This PR also makes sure the user only can use (dynamic) idle using the right setup.

PID Tuning tab:
Screenshot from 2021-11-21 21-58-29

Motors tab:
Screenshot from 2021-11-21 21-50-50

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: locales/en/messages.json
Comment (ctzsnooze) [inline locales/en/messages.json:1798]: This message applies for Transition; jitter reduction is very different and needs a different message.  Maybe:
"Jitter reduction reduces Feedforward when the sticks move slowly.  This allows smooth, jitter-free flight when making smooth slow arcs, yet provides full strong feedforward when the sticks move quickly.  A higher threshold value (10-12) is more useful for cinematic or HD freestyle purposes, and a slightly lower value (5) better for racing or higher speed RC links."

Comment (haslinghuis) [reply locales/en/messages.json]: Updated.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by Asizon** [comment]
@haslinghuis in my opinion if we are thinking on finally release 4.3, this PR should go in to 10.9.0.

**Comment by haslinghuis** [comment]
Think we have time for this as 4.3 is not finished, but agree with code freeze / going into release candidate state. We need a new milestone 10.8.1 I suggest as I like to propose to have minor releases after 10.8 releases.

**Comment by Zuldan** [comment]
Getting a Release Candidate for Xmas would be amazing! Bring on the feature freeze!

**Review by ctzsnooze — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Review by Asizon — APPROVED** [review]
I think this hide disable style doesnt go with our other disabled styles arround tabs, but aproved.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Pull request merged**
